Package.swift

```diff
@@ -14,7 +14,9 @@ let package = Package(
         .package(name: "BigInt", url: "https://github.com/attaswift/BigInt", from: "5.0.0"),
         .package(name: "GenericJSON", url: "https://github.com/zoul/generic-json-swift", from: "2.0.0"),
         .package(url: "https://github.com/GigaBitcoin/secp256k1.swift.git", .upToNextMajor(from: "0.6.0")),
-        .package(url: "https://github.com/OpenCombine/OpenCombine.git", from: "0.13.0")
+        .package(url: "https://github.com/OpenCombine/OpenCombine.git", from: "0.13.0"),
+        .package(url: "https://github.com/vapor/websocket-kit.git", from: "2.0.0"),
```

Comment:
Wondering if we should create different products in package definition to allow importing WSS only if required. What do you think @dnKaratzas ?